utils/misc.ml

```diff
@@ -584,21 +586,28 @@ module Color = struct
     | "error" -> (!cur_styles).error
     | "warning" -> (!cur_styles).warning
     | "loc" -> (!cur_styles).loc
+    | "difference" -> (!cur_styles).difference
     | _ -> raise Not_found
 
   let color_enabled = ref true
+  let marker_enabled = ref false
+  let diff_markers = ref ("!(",")")
```

Comment:
What is the impact of this choice of ASCII representation for diff markers? Am I correct that they will never be shown to the user, only used by the internal Format machinery?

Reply:
They are only visible when requested with the `-diff-marker` option, with the idea that such marker are friendlier to parse than terminal escape sequences (initially, I added them as an accessibility option, but they seem to be a failure on this point)